src/app/api/notes/new/route.ts

```diff
@@ -0,0 +1,37 @@
+import prisma from "@/utils/prisma";
+import { NextRequest, NextResponse } from "next/server";
+
+export async function POST(request: NextRequest) {
+    const userID = 1; // TODO need to be retrieved from the token
+    if (request.body) {
+        try {
+            const body = JSON.parse(await request.text());
+            const { content, noteIV, contentChecksum, creationTimestamp } =
```

Comment:
No type checking, you may want to use an enum here and check if the body has the right values before trying to add it to the DB.